index.html

```diff
@@ -0,0 +1,110 @@
+    </header>
+    <div class="hero"></div>
+    <div class="content">
+      <!-- main section starts here-->
+      <section class="search-engine-optimization banner-titles">
+        <img
+          src="./assets/images/search-engine-optimization.jpg"
+          class="float-left"
+        />
+        <h2>Search Engine Optimization</h2>
+        <p>
+          The dominance of mobile internet use means that users are searching
+          for the right business as they travel, shop, or sit on their couch at
+          home. Search Engine Optimization (SEO) allows you to increase your
+          visibility and find the right customers for your business.
+        </p>
+      </section>
+      <section
+        id="online-reputation-management"
+        class="online-reputation-management"
+      >
+        <img
+          src="./assets/images/online-reputation-management.jpg"
+          class="float-right"
+        />
+
+        <h2>Online Reputation Management</h2>
+        <p>
+          The web is full of opinions, and some of these can be negative. Social
+          media allows anyone with an internet connection to say whatever they
+          want about your business. Online Reputation Management gives you the
+          control over what potential customers see when they search for your
+          business.
+        </p>
+      </section>
+
+      <section id="social-media-marketing" class="social-media-marketing">
+        <img
+          src="./assets/images/social-media-marketing.jpg"
+          class="float-left banner-images"
+        />
+        <h2>Social Media Marketing</h2>
+        <p>
+          Social media continues to have a sizable influence on buying habits.
+          Social media marketing helps you determine which platforms are suited
+          to your brand, using analytics to find the right markets and increase
+          your lead generation.
+        </p>
+      </section>
+    </div>
+    <div class="benefits">
+      <div class="benefit">
+        <h3>Lead Generation</h3>
+        <img src="./assets/images/lead-generation.png" />
+        <p>
+          Inbound strategies for lead generation require less work for your
+          business, bringing customers directly to your website.
+        </p>
+      </div>
+      <div class="benefit">
+        <h3>Brand Awareness</h3>
+        <img src="./assets/images/brand-awareness.png" />
+        <p>
+          Users find your business through paid and organic searches, increasing
+          the search ranking and visibility for your business.
+        </p>
+      </div>
+      <div class="benefit">
+        <h3>Cost Management</h3>
+        <img src="./assets/images/cost-management.png" />
+        <p>
+          As the search ranking for your business increases, your advertising
+          costs decrease, and you no longer need to advertise your page.
+        </p>
+      </div>
+    </div>
+    <footer class="footer">
+      <h2>Made with ❤️️ by Horiseon</h2>
```

Comment:
I'm being a bit picky here, but this `h2` could of been a `h4`. This would of made it flow with the other `h` tags in the code. However, as the footer is it's own section, a `h2` is fine.